src/components/Filters/index.js

```diff
@@ -56,14 +56,18 @@ const SimpleSearchBox = styled.div`
   }
 `
 
-const FilterWrapper = styled(Box)`
-  position: relative;
-`
+const FilterWrapper = styled(Box)`position: relative;`
```

Comment:
I like it when prettier just hasn't yet decided what to use and randomly changes stuff between versions.
It's like its taste evolves with time or maybe it's a kind of a joke messing with us on reviews.